### ISSUE TITLE
RFE: Silent mode

### ISSUE THREAD

**Issue by sils1297** [yegor256/rultor#885]
People are getting a lot of emails in our project. GitHub's bad at regulating that. It would be nice to not have PR's and issues cluttered with unneeded comments. E.g. for a merge I get an email from github on merge anyways that rultor merged that stuff. Further flooding with "hey I'm doing a simple merge now like you requested" and "yep that worked" messages is not needed IMO. Zero messages on success, one message on failure would be my aim.

If I want to see the log one would probably need an additional log command or a parameter to the merge command (verbose=true).

Same probably also applies to releases.

**Comment by yegor256** [comment]
@sils1297 make sense, thanks. how about this:

```yaml
merge:
  quiet: true
```

`false` by default. If it's set to `true`, you see no "success" messages, only when failed.

**Comment by sils1297** [comment]
@yegor256 I'd rather prefer a global setting for all commands plus also avoiding the command acknowledgements. I.e. the merge command should *just* merge with no comments at all.

**Comment by yegor256** [comment]
@sils1297 yeah, make sense. so it will be a global option (`false` by default):

```yaml
quiet: true
```

**Comment by alex-palevsky** [comment]
@sils1297 since there is no milestone yet I set it to "2.0"

**Comment by alex-palevsky** [comment]
@yegor256 this is your task now,help us

**Comment by sils1297** [comment]
@yegor256 just an idea about how you can make it even cooler - provide the information while not doing comments: use the status API to set a rultor PR status.

**Comment by yegor256** [comment]
@alex-palevsky assign someone else pls

**Comment by alex-palevsky** [comment]
> @alex-palevsky assign someone else pls

@yegor256 -30 points to your rating :(

**Comment by alex-palevsky** [comment]
> @alex-palevsky assign someone else pls

@yegor256 got it, someone else will be assigned soon

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear OK, I put "postponed" label here

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear got it, someone else will be assigned soon
